morphocut/stream.py

```diff
@@ -15,9 +15,18 @@
 
 class TQDM(Node):
     """
-    Provide a progress indicator via `tqdm`_.
+    Provide a progress indicator via `tqdm`_. Returns an iterator which prints a dynamically
+    updating progressbar.
 
-    .. _tqdm: https://github.com/tqdm/tqdm
+    ..Note:
```

Comment:
`..note:` has to be lowercase, it is a rst directive: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html